packages/clickhouse-server.postinstall

```diff
@@ -26,6 +26,14 @@ if [ ! -f "/etc/debian_version" ]; then
 fi
 
 if [ "$1" = configure ] || [ -n "$not_deb_os" ]; then
+    # 'ii' means install and installed
+    # ('ic' for installed but configuration incomplete)
+    # If it is the first installation, we might encounter the 'ic' (configuration incomplete) status when executing the postinst script
+    installed=$(dpkg -l | grep -E '^ii\s+clickhouse-server\s')
```

Comment:
We use the same package for all Linux repositories, which means, the usage of `dpkg` is illegal.